### ISSUE TITLE
[FEATURE] Generate Newsletter

### ISSUE THREAD

**Issue by EduardDopler** [DivanteLtd/storefront-ui#927]
**Describe how this feature will help us improve**
Currently we need to scrape the git log manually for our regular newsletter. As discussed in the core team, we want to automate this process.

**Is your proposal related to a problem?**
No.

**Describe the solution you'd considered**
There should be a script that parses the git log between two tags and renders the commits grouped by category (features, fixes, refactorings, breaking changes, other) in a copy & paste-friendly format.

**Additional context**
None.

**Comment by EduardDopler** [comment]
@aniamusial How would you like the following format? I think Markdown would serve us well because we could use it for GitHub directly.

With the script I'm about to finish you could generate this by running:
```
node scripts/create-gitlog @storefront-ui/vue@0.5.0..@storefront-ui/vue@0.6.2
```

The last part (with the two dots `..`) can be anything that git understands as a _revision range_. So the most common case would be two tags but it also understands things like:
* _"last 10 commits"_ -> `node scripts/create-gitlog -10`
* _"everything since last tag"_ -> `node scripts/create-gitlog $(git describe --tags --abbrev=0 @^)..@`
* _"... in reversed order" -> `... --reverse`
* and so on.

-----------

# 💚 Storefront UI Newsletter Mar 21 2020 💚



## ✨ NEW FEATURES
* ([`5f0065bb`](https://github.com/DivanteLtd/storefront-ui/commit/5f0065bb)) feat(example): Thank you page (#742)
* ([`e8ad2c96`](https://github.com/DivanteLtd/storefront-ui/commit/e8ad2c96)) feat: add SfProductCardHorizontal (#716)
* ([`bd505235`](https://github.com/DivanteLtd/storefront-ui/commit/bd505235)) feat(example): error page (#759)
* ([`6df9f810`](https://github.com/DivanteLtd/storefront-ui/commit/6df9f810)) feat: add font-display: swap rule to google fonts (#771)
* ([`da1a32ce`](https://github.com/DivanteLtd/storefront-ui/commit/da1a32ce)) feat: add v-focus directive (#753)
* ([`efb079ee`](https://github.com/DivanteLtd/storefront-ui/commit/efb079ee)) feat: pull request template update (#749)
* ([`486a1599`](https://github.com/DivanteLtd/storefront-ui/commit/486a1599)) feat: add quantity to cart icon in SfHeader (#734)
* ([`c708062b`](https://github.com/DivanteLtd/storefront-ui/commit/c708062b)) feat: add default aria-label to SfQuantitySelector (#735)
* ([`cbb59c31`](https://github.com/DivanteLtd/storefront-ui/commit/cbb59c31)) feat: add required slots to SfTabs (#736)
* ([`9241efb5`](https://github.com/DivanteLtd/storefront-ui/commit/9241efb5)) feat: add hasCarousel prop for mobile SfGroupedProduct (#715)
* ([`58d67f1d`](https://github.com/DivanteLtd/storefront-ui/commit/58d67f1d)) feat: add stylelint for .scss in shared packages (#732)
* ([`d362cf68`](https://github.com/DivanteLtd/storefront-ui/commit/d362cf68)) feat: add isMobile observer (#701)
* ([`3584689f`](https://github.com/DivanteLtd/storefront-ui/commit/3584689f)) feat: add SfBadge to SfProducCard (#693)

## 💥 BREAKING CHANGES
* ([`15a6fcd5`](https://github.com/DivanteLtd/storefront-ui/commit/15a6fcd5)) BREAKING CHANGE: adjust css variable to default pattern (#744)
* ([`a06d9aca`](https://github.com/DivanteLtd/storefront-ui/commit/a06d9aca)) BREAKING CHANGE: replace SCSS variable to CSS variable (#673)

## 🐛 FIXES
* ([`2c1059ff`](https://github.com/DivanteLtd/storefront-ui/commit/2c1059ff)) fix: ssr in SfScrollable (#855)
* ([`53322009`](https://github.com/DivanteLtd/storefront-ui/commit/53322009)) fix: ssr error in SfHeader(#854)
* ([`3d1f1d58`](https://github.com/DivanteLtd/storefront-ui/commit/3d1f1d58)) fix: badge icon radius (#811)
* ([`9f622edb`](https://github.com/DivanteLtd/storefront-ui/commit/9f622edb)) fix: title props name in SfSidebar (#810)
* ([`8f2c01ae`](https://github.com/DivanteLtd/storefront-ui/commit/8f2c01ae)) fix: a11y for SfTab component (#787)
* ([`8c5e8440`](https://github.com/DivanteLtd/storefront-ui/commit/8c5e8440)) fix: missing hasCarousel prop in stories (#798)
* ([`1dcb2ba3`](https://github.com/DivanteLtd/storefront-ui/commit/1dcb2ba3)) fix: add transparent bcg to SfButton outline modifiers (#791)
* ([`df1ad2f7`](https://github.com/DivanteLtd/storefront-ui/commit/df1ad2f7)) fix: too bright hover color on SfButton(#793)
* ([`57b07712`](https://github.com/DivanteLtd/storefront-ui/commit/57b07712)) fix: add openTab in SfTabs stories (#789)
* ([`f0f84020`](https://github.com/DivanteLtd/storefront-ui/commit/f0f84020)) fix: add type to show-password button (#786)
* ([`cc7e7903`](https://github.com/DivanteLtd/storefront-ui/commit/cc7e7903)) fix: position of SfProductCard add to card button (#779)
* ([`a89b96a8`](https://github.com/DivanteLtd/storefront-ui/commit/a89b96a8)) fix: set initial slide in SfGallery (#760)
* ([`0bc6ac0d`](https://github.com/DivanteLtd/storefront-ui/commit/0bc6ac0d)) fix: small ssr bugs (#776)
* ([`3601b27f`](https://github.com/DivanteLtd/storefront-ui/commit/3601b27f)) fix: docs for CSS custom properties (#775)
* ([`b15eea45`](https://github.com/DivanteLtd/storefront-ui/commit/b15eea45)) fix: overlay transition opacity conflicting (#773)
* ([`d21c8f7e`](https://github.com/DivanteLtd/storefront-ui/commit/d21c8f7e)) fix: add variable for z-index (#725)
* ([`e39fb8ee`](https://github.com/DivanteLtd/storefront-ui/commit/e39fb8ee)) fix: docs build fails due to outdated links (#751)
* ([`362fc233`](https://github.com/DivanteLtd/storefront-ui/commit/362fc233)) fix: Add aria-label in SfCircleIcon stories (#727)
* ([`b995ae01`](https://github.com/DivanteLtd/storefront-ui/commit/b995ae01)) fix: a lot imports of global CSS variables (#731)
* ([`05b5ca6b`](https://github.com/DivanteLtd/storefront-ui/commit/05b5ca6b)) fix: double pictures is not adjusted to size (#683)
* ([`8422bc05`](https://github.com/DivanteLtd/storefront-ui/commit/8422bc05)) fix: prevent glide mount for empty container (#694)
* ([`9c2b9c4a`](https://github.com/DivanteLtd/storefront-ui/commit/9c2b9c4a)) fix(ui): cheating lerna
* ([`554c6d73`](https://github.com/DivanteLtd/storefront-ui/commit/554c6d73)) fix(ui): rever sfui css
* ([`01b24d17`](https://github.com/DivanteLtd/storefront-ui/commit/01b24d17)) fix(ui): sfui css

## ♻️ REFACTOR
* ([`08fca6a1`](https://github.com/DivanteLtd/storefront-ui/commit/08fca6a1)) refactor: remove sfui.scss file (#794)
* ([`7f0475aa`](https://github.com/DivanteLtd/storefront-ui/commit/7f0475aa)) refactor: SfModal and SfSidebar on mobile (#755)
* ([`17aefa55`](https://github.com/DivanteLtd/storefront-ui/commit/17aefa55)) refactor: updated SfColor stories (#774)
* ([`de923891`](https://github.com/DivanteLtd/storefront-ui/commit/de923891)) refactor: change is-active and is-disabled to modifiers (#770)
* ([`46fb4f26`](https://github.com/DivanteLtd/storefront-ui/commit/46fb4f26)) refactor: remove SfOptions component (#769)
* ([`421ba48e`](https://github.com/DivanteLtd/storefront-ui/commit/421ba48e)) refactor: remove calc from SfChevron (#758)
* ([`fc6f2f8e`](https://github.com/DivanteLtd/storefront-ui/commit/fc6f2f8e)) refactor: adjust SfFooter to SfHeader and MegaMenu (#741)
* ([`50f18938`](https://github.com/DivanteLtd/storefront-ui/commit/50f18938)) refactor: chang logic for showing content in SfTabs (#738)
* ([`7134e418`](https://github.com/DivanteLtd/storefront-ui/commit/7134e418)) refactor: remove all unused --secondary modifiers (#691)
* ([`d40be4bb`](https://github.com/DivanteLtd/storefront-ui/commit/d40be4bb)) refactor: add isSticky props to SfHeader (#739)
* ([`13382bd0`](https://github.com/DivanteLtd/storefront-ui/commit/13382bd0)) refactor: clean organisms stories (#707)
* ([`c7a82ce6`](https://github.com/DivanteLtd/storefront-ui/commit/c7a82ce6)) refactor: replace camelCase slot names with kebab-case (#717)
* ([`c5f6d536`](https://github.com/DivanteLtd/storefront-ui/commit/c5f6d536)) refactor: remove usage of yarpm and require yarn (#730)
* ([`d1fa6988`](https://github.com/DivanteLtd/storefront-ui/commit/d1fa6988)) refactor: remove SfCounter component (#705)
* ([`8cb79dc6`](https://github.com/DivanteLtd/storefront-ui/commit/8cb79dc6)) refactor: add label to SfBottomNavigation (#706)
* ([`568f6e42`](https://github.com/DivanteLtd/storefront-ui/commit/568f6e42)) refactor: remove min from SfQuantitySelector (#720)

## 📝 DOCS
* ([`25da463e`](https://github.com/DivanteLtd/storefront-ui/commit/25da463e)) docs: reorder core team members
* ([`1ec9a24d`](https://github.com/DivanteLtd/storefront-ui/commit/1ec9a24d)) docs: updated github labeling guide (#785)
* ([`a9929a20`](https://github.com/DivanteLtd/storefront-ui/commit/a9929a20)) docs: update github guideliness (#784)
* ([`7f90034c`](https://github.com/DivanteLtd/storefront-ui/commit/7f90034c)) docs: update installation guide (#778)
* ([`c0eb91d9`](https://github.com/DivanteLtd/storefront-ui/commit/c0eb91d9)) docs: add Storefront UI gitflow (#756)
* ([`18ef34fc`](https://github.com/DivanteLtd/storefront-ui/commit/18ef34fc)) docs: addjust docs to CSS custom properties (#748)
* ([`5728e2ee`](https://github.com/DivanteLtd/storefront-ui/commit/5728e2ee)) docs: add Marta Radziszewska to core team (#762)
* ([`0fb6fafb`](https://github.com/DivanteLtd/storefront-ui/commit/0fb6fafb)) docs: update contributing guide (#745)
* ([`671fd685`](https://github.com/DivanteLtd/storefront-ui/commit/671fd685)) docs: add colors palette and sizes to documentation (#609)

## 🔔 Other Noteworthy Goodies
* ([`1a90e48b`](https://github.com/DivanteLtd/storefront-ui/commit/1a90e48b)) chore: small improvements in examples (#796)
* ([`b0294e11`](https://github.com/DivanteLtd/storefront-ui/commit/b0294e11)) chore: rename ReviewOrder to ConfirmOrder (#695)
* ([`21b7e368`](https://github.com/DivanteLtd/storefront-ui/commit/21b7e368)) chore: remove unused nuxt package (#692)
* ([`2d984f53`](https://github.com/DivanteLtd/storefront-ui/commit/2d984f53)) test(unit): fixing tests for component with provide/inject pattern (#662)
* ([`397a1ced`](https://github.com/DivanteLtd/storefront-ui/commit/397a1ced)) test: add render test to missed components (#644)

**Comment by aniamusial** [comment]
@EduardDopler wow, what a great work you've done! I love the new markdown, I'll check the PR later in the day, but I can already say thath I'm IMPRESSED! :) Thank you!
